soccer/KickEvaluator.cpp

```diff
@@ -338,44 +335,43 @@ vector<tuple<float, float> > KickEvaluator::convert_robots_to_polar(
     return botLocations;
 }
 
-void KickEvaluator::init_gradient_configs(ParallelGradient1DConfig* pConfig,
-                                          KickEvaluatorArgs* keArgs) {
-    pConfig->GA1DConfig.reserve(keArgs->robotStDevs.size());
+void KickEvaluator::init_gradient_configs(
+    ParallelGradient1DConfig* pConfig,
+    function<tuple<float, float>(float)>* func, const vector<float>* robotMeans,
+    const vector<float>* robotStDevs, const float boundaryLower,
```

Comment:
I'd recommend you change those const pointers to const references. Its a good coding guideline we're been following in the rest of soccer (trying to at least) and prevents null from being passed in (unless you really want to be able ot pass in null). Pointers for non const stuff is fine.